OpenRA.Mods.Common/Traits/CombatDebugOverlay.cs

```diff
@@ -53,10 +59,11 @@ public void RenderAfterWorld(WorldRenderer wr, Actor self)
 			if (healthInfo != null)
 				healthInfo.Shape.DrawCombatOverlay(wr, wcr, self);
 
-			if (blockInfo != null)
+			var block = blocker.Where(Exts.IsTraitEnabled).ToList();
+			if (block.Any())
```

Comment:
Maybe `blocker` -> `allBlockers` and `block` -> `blockers` for consistency with elsewhere in your PR where `blockers` are the enabled `IBlocksProjectiles` traits.